apps/mobile/src/libs/packages/notification/notification.package.ts

```diff
@@ -0,0 +1,37 @@
+import Toast from "react-native-toast-message";
+
+import {
+	DEFAULT_ERROR_MESSAGES,
+	DEFAULT_ERROR_TITLE,
+} from "./libs/constants/constant";
+
+class Notification {
+	public error(
+		title: string = DEFAULT_ERROR_TITLE,
+		text: string = DEFAULT_ERROR_MESSAGES,
+	) {
+		Toast.show({
+			text1: title,
```

Comment:
Let's create a ToastMessageTitle enum with titles for each type of toast and use it as a default title for each one

Reply:
I create create a ToastMessageTitle enum. What default value do you expect in DEFAULT_TOAST_MESSAGE = { title: '', message: ''}, or should it be empty?

Reply:
Now I'm thinking about creating the default message only for error toast (title: ToastMessageTitle.ERROR, message: DEFAULT_ERROR_MESSAGE) and we don't need DEFAULT_ERROR_TITLE because it will come from enum.

It should look like this. 
`public info({ title, message }: IToastMessageOptions): void {
		this.showToast({
			type: ToastMessageType.INFO,
			title: title ?? ToastMessageTitle.INFO,
			message,
		});
	}`